fay-base/fay-base.cabal

```diff
@@ -1,5 +1,5 @@
 name:                fay-base
```

Comment:
This file doesn't need to be updated since the changes don't affect fay-base (and assuming we don't need to update fay's version to 0.24)